composer.json

```diff
@@ -15,10 +15,12 @@
         "psr-4": {"Matomo\\ReportingApi\\": "src/"}
     },
     "require": {
-        "guzzlehttp/guzzle": "~6.0"
+        "php": "^7.4|^8.0",
+        "guzzlehttp/guzzle": "~7",
+        "phpspec/prophecy-phpunit": "^2.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.0|^6.0|^7.0",
+        "phpunit/phpunit": ">5.0",
```

Comment:
The minimum PHPUnit version supporting PHP 7.3 is 7. See https://phpunit.de/supported-versions.html. We have to support PHPUnit 7,  8, 9